hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/volume/StorageVolume.java

```diff
@@ -459,7 +461,7 @@ protected StorageLocationReport.Builder reportBuilder() {
     StorageLocationReport.Builder builder = StorageLocationReport.newBuilder()
         .setFailed(isFailed())
         .setId(getStorageID())
-        .setStorageLocation(volumeRoot)
+        .setStorageLocation(storageDirStr)
```

Comment:
Can use storageDir.getAbsolutePath() directly here.

Reply:
`getAbsolutePath()` involves filesystem lookup, I wanted to avoid that.

Reply:
Please add a comment for field storageDirStr about the purpose.

Reply:
Please add a comment for field storageDirStr for this motivation.